egs/librispeech/ASR/lstm_transducer_stateless/lstm.py

```diff
@@ -209,7 +218,13 @@ def forward(
         # lengths = ((x_lens - 3) // 2 - 1) // 2 # issue an warning
         #
         # Note: rounding_mode in torch.div() is available only in torch >= 1.8.0
-        lengths = (((x_lens - 3) >> 1) - 1) >> 1
+        if not self.is_pnnx:
+            lengths = (((x_lens - 3) >> 1) - 1) >> 1
```

Comment:
so // (integer division) not supported either in ncnn?

Reply:
No, ncnn's binary op only supports floating point operation.
Also, `>>` the right shift operator is not supported either in ncnn.
(Note: we can extend ncnn to support `>>` if there is a need.)